### DISCUSSION TITLE
Alpha, add new menu entry

### DISCUSSION THREAD

**Discussion by lexaone** [LunarVim/LunarVim#3074] [Q&A] [answered]
Hello, it is possible to add custom menu entry to dashboard menu?

Previously, I just add something like this in config.lua:

table.insert(lvim.builtin.alpha.dashboard.section.buttons.entries,
   { "q", "Q Quit NVIM", ":qa<CR>" })

and viola - I have additional entry in menus. 
But recently something changed with alpha plugin inside lunarvim - and now my code doesn't work.
Unfortunately, I doesn't know lua so much to understand, how to change it to work again.
(I'm using rolling version of LunarVim)

**Comment by lexaone** [comment, marked as answer]
after some digging in Lvim sources I have answer to my question:
to add additional menu entry in dashboard u need to add following code in lvim/config.lua:

```
local dashboard = require "alpha.themes.dashboard"
table.insert(lvim.builtin.alpha.dashboard.section.buttons.val, dashboard.button("q", "  Quit NVIM", ":qa<CR>"))

```

**Reply by MichaelOren** [reply]
I added this to open up the previous session

```
local dashboard = require "alpha.themes.dashboard"
table.insert(lvim.builtin.alpha.dashboard.section.buttons.val,
  dashboard.button("s", "  Open Last Session", "<cmd>lua require('persistence').load()<cr>"))
```
(I did not change your icon because I don't know where to get the icons from 😂 )

Thanks for your help!

**Reply by cassepipe** [reply]
@MichaelOren Does that still work for the latest stable release for you ?
I got this error :

```
[WARN ] lvim: "Invalid configuration: /home/cassepipe/.config/lvim/config.lua:55: bad argument #1 to 'insert' (table expected, got nil)" file="[C]", line=-1
```

Seems like  `lvim.builtin.alpha.dashboard.section.buttons.val` is `nil`

**Reply by lexaone** [reply]
@cassepipe 
try this:
```
table.insert(lvim.builtin.alpha.dashboard.section.buttons.entries,
  { "SPC jdc", "  DearDiary Today Entry", ":DearDiaryToday<CR>" })
table.insert(lvim.builtin.alpha.dashboard.section.buttons.entries,
  { "s", "  Open Last Session", "<cmd>lua require('persistence').load()<cr>" })
```
this works again for me at least for master version

**Reply by cassepipe** [reply]
Oh crap I was pretty sure I had tried that and it didn't work  ! (I explored what was there thanks to to autocompletion as suggested per the docs)
Thanks a lot.
